lib/Classifiers/Audio/MusicnnClassifier.php

```diff
@@ -7,116 +7,76 @@
 
 namespace OCA\Recognize\Classifiers\Audio;
 
+use OCA\Recognize\Classifiers\Classifier;
+use OCA\Recognize\Db\Audio;
+use OCA\Recognize\Db\AudioMapper;
 use OCA\Recognize\Service\Logger;
 use OCA\Recognize\Service\TagManager;
-use OCP\Files\File;
+use OCP\DB\Exception;
+use OCP\Files\IRootFolder;
 use OCP\IConfig;
-use OCP\Files\InvalidPathException;
-use OCP\Files\NotFoundException;
 use Psr\Log\LoggerInterface;
-use Symfony\Component\Process\Exception\ProcessTimedOutException;
-use Symfony\Component\Process\Exception\RuntimeException;
-use Symfony\Component\Process\Process;
 
-class MusicnnClassifier {
+class MusicnnClassifier extends Classifier {
 	public const FILE_TIMEOUT = 40; // seconds
 	public const FILE_PUREJS_TIMEOUT = 300; // seconds
 	public const MODEL_DOWNLOAD_TIMEOUT = 180; // seconds
+	public const MODEL_NAME = 'musicnn';
 
 
 	private LoggerInterface $logger;
 
 	private TagManager $tagManager;
 
 	private IConfig $config;
+	/**
+	 * @var \OCP\Files\IRootFolder
+	 */
+	private IRootFolder $rootFolder;
+	/**
+	 * @var \OCA\Recognize\Db\AudioMapper
+	 */
+	private AudioMapper $audioMapper;
```

Comment:
```suggestion
	private LoggerInterface $logger;
	private TagManager $tagManager;
	private IConfig $config;
	private IRootFolder $rootFolder;
	private AudioMapper $audioMapper;
```

No need for phpdoc with the new PHP type annotation ;)